src/main/interface/settings.c

```diff
@@ -351,6 +351,7 @@ const lookupTableEntry_t lookupTables[] = {
     { lookupTableGyroOverflowCheck, sizeof(lookupTableGyroOverflowCheck) / sizeof(char *) },
 #endif
     { lookupTableRatesType, sizeof(lookupTableRatesType) / sizeof(char *) },
+
```

Comment:
This whitespace is not necessary, none of preprocessor macros above has it :)

Reply:
Will remove